extensions/amp-form/0.1/test/test-form-validators.js

```diff
@@ -123,22 +124,78 @@ describes.realWin('form-validators', {amp: true}, env => {
     });
   });
 
+  describe('FormValidator', () => {
+    let form, validator;
+
+    beforeEach(() => {
+      const doc = env.win.document;
+      form = getForm(doc);
+      validator = new FormValidator(form);
```

Comment:
Looks weird to me since this is technically an abstract class but it works because Javascript. Feel free to swap for an implementation class or disregard because it doesn't practically matter.